ephypype/import_ctf.py

```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
         raw.save(raw_fif_file)
     else:
         print(('*** RAW FIF file %s exists!!!' % raw_fif_file))
-        
+
     return raw_fif_file
 
 
-#def convert_ds_to_raw_fif(ds_file):
+# def convert_ds_to_raw_fif(ds_file):
 
-	#import os
+    #import os
 
-	#from nipype.utils.filemanip import split_filename as split_f
+    #from nipype.utils.filemanip import split_filename as split_f
 
-	#subj_path,basename,ext = split_f(ds_file)
+    #subj_path,basename,ext = split_f(ds_file)
 
-	
-	##basename = os.path.splitext(ds_file)[0]
+    ##basename = os.path.splitext(ds_file)[0]
 
-	#print subj_path,basename,ext
-	##0/0
+    # print subj_path,basename,ext
+    # 0/0
 
-	#raw_fif_file = os.path.join(subj_path,basename + "_raw.fif")
+    #raw_fif_file = os.path.join(subj_path,basename + "_raw.fif")
 
-	#os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,ds_file) + " --fif " + raw_fif_file)
+    #os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,ds_file) + " --fif " + raw_fif_file)
 
-	#return raw_fif_file
+    # return raw_fif_file
```

Comment:
same here. If you have comments that are just code, it should probably be removed